guides/release/accessibility/application-considerations.md

```diff
@@ -1,59 +1,59 @@
 
-There are some existing Ember addons that may help you make your app more accessible. Each addon should be evaluated for its own usefulness and merit- you may find in some instances, that it would be better to implement the ideas presented in the addon in your own application.
+Il existe déjà des addons Ember pouvant vous aider à rendre une application plus accessible.
+Chaque addon doit être choisi en fonction des bénéfices qu'il apporte et de son utilité. Il est possible que dans certains cas, il soit préférable de réimplémenter vous-même une partie des fonctionnalités d'un addon, directement dans votre application.
 
-Here are some examples of accessibility-focused addons created by many people throughout the Ember community:
+Voici quelques exemples d'addons axés sur l'accessibilité créés par des membres de la communauté Ember:
 
-- [ember-a11y-landmarks](https://github.com/ember-a11y/ember-a11y-landmarks) - Ember addon to help with landmark roles for better accessibility
-- [ember-component-focus](https://github.com/ember-a11y/ember-component-focus) - A mixin for adding methods to your Ember components that help you manage the currently focused element.
-- [ember-gestures](https://github.com/html-next/ember-gestures) - Ember Gestures provides an easy way to use gestures by making it simple to define and use HammerJS managers and recognizers throughout your app.
-- [ember-steps](https://github.com/rwjblue/ember-steps) - Declarative create wizards, tabbed UIs, and more
-- [ember-page-title](https://github.com/tim-evans/ember-page-title) - Page title management for Ember.js Apps
-- [ember-self-focused](https://github.com/linkedin/self-focused/tree/master/packages/ember-self-focused) - Focus on route on transition
-- [ember-keyboard](https://github.com/patience-tema-baron/ember-keyboard) - An Ember.js addon for the painless support of keyboard events
-- [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) - A suite of accessibility tests that can be run within the Ember testing framework
-- [a11y-announcer](https://github.com/ember-a11y/a11y-announcer) - An accessible ember route change announcer
-- [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) - linter for Ember templates
-  ![Template Linting Preview](/images/accessibility/application-considerations/template-lint.png)
+- [ember-a11y-landmarks](https://github.com/ember-a11y/ember-a11y-landmarks) - Un addon Ember facilitant l'utilisation des rôles sémantique HTML afin d'améliorer l'accessibilité.
+- [ember-component-focus](https://github.com/ember-a11y/ember-component-focus) - Un mixin pour ajouter des méthodes à vos composants Ember afin de vous aider à gérer l'élément ayant le focus
+- [ember-gestures](https://github.com/html-next/ember-gestures) - Ember Gestures fournit un a moyen simple d'utiliser les interactions tactiles en facilitant la définition des contrôler et des reconnaisseurs HammerJS dans une application.
```

Comment:
If someone has a better translation for recognizers, I'm not a big fan of *reconnaisseurs* 😅